MVP/Resources/Translations.de.resx

```diff
@@ -0,0 +1,456 @@
+  </data>
+  <data name="setting_contributionfromurl_title" xml:space="preserve">
+    <value>Beitrag aus URL erstellen</value>
+  </data>
+  <data name="setting_language_description" xml:space="preserve">
+    <value>Standardmässig wird die Sprache ausgewählt, die auf dem Gerät verwendet wird, sofern Übersetzungen dafür verfügbar sind. Wenn du diese App jedoch in einer anderen Sprache verwenden möchtest, kannst du sie hier ändern.</value>
+  </data>
+  <data name="setting_language_title" xml:space="preserve">
+    <value>Sprache</value>
+  </data>
+  <data name="splash_contributionareas" xml:space="preserve">
+    <value>Laden der Beitragsbereiche...</value>
+  </data>
+  <data name="splash_contributiontypes" xml:space="preserve">
+    <value>Synchronisieren der Beitragstypen...</value>
+  </data>
+  <data name="tabs_activities" xml:space="preserve">
+    <value>Aktivitäten</value>
+  </data>
+  <data name="tabs_badges" xml:space="preserve">
+    <value>Abzeichen</value>
+  </data>
+  <data name="tabs_settings" xml:space="preserve">
+    <value>Einstellungen</value>
+  </data>
+  <data name="tabs_statistics" xml:space="preserve">
+    <value>Statistik</value>
+  </data>
+  <data name="themepicker_title" xml:space="preserve">
+    <value>App Modus</value>
+  </data>
+  <data name="theme_dark" xml:space="preserve">
+    <value>Dunkel</value>
+  </data>
+  <data name="theme_light" xml:space="preserve">
+    <value>Hell</value>
+  </data>
+  <data name="theme_systemdefault" xml:space="preserve">
+    <value>Standard</value>
+  </data>
+  <data name="validation_variablefield" xml:space="preserve">
+    <value>{0} ist ein Pflichtfeld.</value>
+  </data>
+  <data name="view_licenses" xml:space="preserve">
+    <value>Lizenzen ansehen</value>
+  </data>
+  <data name="visibility_title" xml:space="preserve">
+    <value>Sichtbarkeit</value>
+  </data>
+  <data name="warning_description_activitylocked" xml:space="preserve">
+    <value>Diese Aktivität wurde im vorherigen Beitragsfenster aufgezeichnet und kann daher nicht bearbeitet oder gelöscht werden.</value>
+  </data>
+  <data name="warning_title" xml:space="preserve">
+    <value>Einen Moment bitte...</value>
+  </data>
+  <data name="warning_title_activitylocked" xml:space="preserve">
+    <value>Aktivität gesperrt</value>
+  </data>
+  <data name="yes" xml:space="preserve">
+    <value>Ja</value>
+  </data>
+  <data name="splash_visibilities" xml:space="preserve">
+    <value>Anzeigen einrichten...</value>
+  </data>
+  <data name="splash_gettingready" xml:space="preserve">
+    <value>Vorbereitungen sind im Gange...</value>
+  </data>
+  <data name="sign_in" xml:space="preserve">
+    <value>Einloggen</value>
+  </data>
+  <data name="swipe_to_continue" xml:space="preserve">
+    <value>Swipe to continue</value>
+  </data>
+  <data name="try_with_demo_account" xml:space="preserve">
+    <value>Try with a demo account</value>
```

Comment:
Mit einem Demo Konto ausprobieren.

Reply:
This might be a bit too wordy, for context:

![image](https://user-images.githubusercontent.com/2419439/103635474-daade080-4f48-11eb-9f27-99f533a0cc7b.png)

Reply:
Mit Demo Konto testen

Reply:
Mit einem Demokonto ausprobieren worked like a charm :wink: